assets/js/analyzer.js

```diff
@@ -24,7 +23,7 @@ if(modeEnabled('analyzation')) {
         });
 
         $('#morphAnalyzerInput').keydown(function (e) {
-            if(e.keyCode === ENTER_KEY_CODE && !e.shiftKey) {
+            if(e.keyCode === config.ENTER_KEY_CODE && !e.shiftKey) {
```

Comment:
This shouldn't be configurable...

Reply:
What constants shall I move back to JS?

Reply:
I think you can figure out the correct set.